components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenuBuilder.kt

```diff
@@ -12,7 +12,8 @@ import android.content.Context
  * @param items List of BrowserMenuItem objects to compose the menu from.
  */
 class BrowserMenuBuilder(
-    val items: List<BrowserMenuItem>
+    val items: List<BrowserMenuItem>,
+    val isCustomTabMenu: Boolean? = false
```

Comment:
I think this is too specific. If apps use menus in different situations then we can't add booleans here for all possible menus that an app may use. To make this more generic we could allow passing in an additional map of values that will be added as extras. Would that work?

Reply:
Agreed, updated to consider this :)

Reply:
Like that! :) Can you add a KDoc for the parameter explaining what those extras are used for? :)